fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js

```diff
@@ -62,7 +62,7 @@ class NavSystem extends BaseInstrument {
         if (this.nodeName.includes('CDU')) {
             this.currFlightPhaseManager = Fmgc.getFlightPhaseManager();
 
-            this.currFlightPlanService = new Fmgc.FlightPlanService();
+            this.currFlightPlanService = new Fmgc.FlightPlanService(new EventBus());
```

Comment:
Am I correct that this change is only relevant for the A380? I don't see how passing in a new event bus instance in the constructor is different from just instantiating it in the constructor of the flight plan service.

Reply:
Yep, this is only relevant for the A380.

Reply:
There should only really be one event bus for each entire instrument unless you're trying to do something very specific.